VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -592,9 +599,14 @@ Function Get-PasswordPolicyDefault {
 
     Param (
         [Parameter (Mandatory = $false, ParameterSetName = 'json')] [ValidateNotNullOrEmpty()] [Switch]$generateJson,
+        [Parameter (Mandatory = $true)] [ValidateSet('4.4.0','4.5.1','5.0.0')] [String]$version='5.0.0',
```

Comment:
Are we sure we want to default to 5.0.0 or require the version to be set?

Reply:
Yes. Gary confirmed default has to be set and has to be 5.0.0